index.html

```diff
@@ -47,11 +59,38 @@ <h1 class="headline-1-h1">I’m Mariane
             </ul>
         </div>
     </section>
+    <!--headline section Desktop View-->
+    <section class="desktop_headline">
+        <div class="desktop_headline-1">
+            <h1 class="headline-1-h1">I’m Alexandra<br> Glad to see you!
+            </h1>
+        </div>
+        <div class="headline-2">
+            <p class="headline-2-p">I’m a software developer! I can help you build
+                 a product , feature or website Look through some of my work and 
+                 experience! If you like what you see and have a project you need coded,
+                  don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="headline-3">
+            <p class="headline-3-p">LET’S CONNECT</p>
+        </div>
```

Comment:
- [ ] Good job adding a proper header on your web page, :clap: , I’ve noticed that there is some overflow with the `headline-2` text , where you are working on screen that have `768px` , Kindly make sure you fix it :+1: , it will make your web page more responsive :+1: 


[Screencast from 02-25-2023 12_05_10 AM.webm](https://user-images.githubusercontent.com/80458199/221303271-c693447c-55ec-4b16-b530-55b7c80b1013.webm)